scripts/deploy.js

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+
+const AWS = require('aws-sdk');
+AWS.config.update({region: process.env.AWS_REGION || 'us-west-1'});
+
+let file;
+try {
+  file = fs.readFileSync(__dirname + '/../function.zip');
+} catch (err) {
+  console.error('Could not read function.zip! ' + err.message + '. Please ensure that you have built babel-bot.');
+  process.exit(1);
+}
+
+console.log('Deploying...');
+const lambda = new AWS.Lambda({apiVersion: '2015-03-31'});
+lambda.updateFunctionCode({
+  FunctionName: 'BabelBot',
+  Publish: true,
+  ZipFile: file,
+}, function(err, data) {
+  if (err) {
+    console.error('Could not update: ' + err.message);
+    process.exit(2);
```

Comment:
There's already an exit code of 1 on line 13... usually you use different exit codes to indicate different conditions.